js/wp-autoupdates.js

```diff
@@ -1,286 +1,88 @@
-
-		// Show loading status.
-		$parent.find( '.theme-autoupdate-label' ).html( '<span class="dashicons dashicons-update spin"></span> ' + wp_autoupdates.enabling );
-
-		$.post(
-			ajaxurl,
-			{
-				action: 'enable_auto_updates',
-				_ajax_nonce: href._wpnonce,
-				type: 'theme',
-				asset: href.theme
-			},
-			function (response) {
-
-			}
-		)
-		.done(function (response) {
-			if ( response.success ) {
-				$parent.html( response.data.return_html );
-				$parent.find('.theme-autoupdate-disable').focus();
-				wp.a11y.speak( wp_autoupdates.auto_enabled, 'polite' );
-			} else {
-				var errorHTML = add_error_notice( html, response.data.error );
-				wp.a11y.speak( response.data.error, 'polite' );
-				$parent.html( errorHTML );
-			}
-			
-		})
-		.fail(function (response) {
-			var errorHTML = add_error_notice( html, wp_autoupdates.auto_update_error );
-			wp.a11y.speak( wp_autoupdates.auto_update_error, 'polite' );
-			$parent.html( errorHTML );
-		})
-		.always(function (response) {
-		});
-	});
-});
\ No newline at end of file
+		label.text( 'enable' === action ? wp_autoupdates.enabling : wp_autoupdates.disabling );
+		anchor.find( '.dashicons-update' ).removeClass( 'hidden' );
+
+		data = {
+			action: 'toggle_auto_updates',
+			_ajax_nonce: href._wpnonce,
+			state:  action,
+			type: type,
+			asset: 'plugin' === type ? href.plugin : href.theme,
+		};
+
+		$.post( ajaxurl, data )
+			.done( function( response ) {
+				if ( response.success ) {
+					// Update the counts in the enabled/disabled views if on on
+					// screen with a list table.
+					// TODO: If either count started out 0 the appropriate span won't
+					//       be there and hence won't be updated.
+					if ( 'themes' !== pagenow ) {
+						var enabled        = $( '.autoupdate_enabled span' ),
+							disabled       = $( '.autoupdate_disabled span' ),
+							enabledNumber  = parseInt( enabled.text().replace( /[^\d]+/g, '' ) ) || 0,
+							disabledNumber = parseInt( disabled.text().replace( /[^\d]+/g, '' ) ) || 0;
+
+						switch ( action ) {
+							case 'enable':
+								++enabledNumber;
+								--disabledNumber;
+
+								break;
+							case 'disable':
+								--enabledNumber;
+								++disabledNumber;
+								break;
+						}
+
+						enabledNumber  = Math.max( 0, enabledNumber );
+						disabledNumber = Math.max( 0, disabledNumber );
+
+						enabled.text( '(' + enabledNumber + ')' );
+						disabled.text( '(' + disabledNumber + ')' );
+					}
+
+					if ( 'enable' === action ) {
+						anchor.removeClass( 'enable' ).addClass( 'disable' );
+						label.text( wp_autoupdates.disable );
+						parent.find( '.auto-update-time').removeClass( 'hidden' );
+					} else {
+						anchor.removeClass( 'disable' ).addClass( 'enable' );
+						label.text( wp_autoupdates.enable );
+						parent.find( '.auto-update-time').addClass( 'hidden' );
+					}
+
+					wp.a11y.speak( 'enable' === action ? wp_autoupdates.enabled : wp_autoupdates.disabled, 'polite' );
```

Comment:
`polite` is redundant, as it's the default value.

Reply:
@audrasjb What are your thoughts here?

Personally, I think it is sometimes **useful** to be explicit with params, even when using the default value...and think this is one of those cases.